catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/content/ContentComponentResolver.java

```diff
@@ -20,8 +20,6 @@
 import org.slf4j.LoggerFactory;
```

Comment:
I guess not, if we aren't using `LOGGER`. I'll remove that and the logger variables